spec/requests/api/drops_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+
+RSpec.describe Api::DropsController, :vcr, type: :request do
+  context 'with several drops present' do
+  let!(:drops) { FactoryGirl.create_list(:drop, 3) }
+
+    describe "GET /drops" do
+
+      it "displays list of  drops in database" do
+
```

Comment:
Just a minor one: I'd suggest dropping the unnecessary whitespace lines (L2 and L11) to keep the house clean.